train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java

```diff
@@ -8,6 +8,11 @@ public class TrainControllerImpl implements TrainController {
 	private int referenceSpeed = 0;
 	private int speedLimit = 0;
 
+	public TrainControllerImpl() {
+		Thread th = new RunThread();
+		th.start();
```

Comment:
Jó-jó, de jobb lenne, ha Cicagombra indulna...

Reply:
A kompatiblitás így is meg van őrizve :handshake: